install/helm-repo/argocd-agent-agent/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for argocd-agent-agent.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Override Namespace to deploy your agent
+namespaceOverride: ""
+
+# Secret names for argo-agent deployment
+tlsSecretName: "argocd-agent-client-tls"
+userPasswordSecretName: "argocd-agent-agent-userpass"
+image: "ghcr.io/argoproj-labs/argocd-agent/argocd-agent"
+imageTag: "latest"
+
+# config-map to config parameters for argocd-agent
+agentMode: "autonomous"
+auth: "mtls:any"
+logLevel: "info"
+server: "http://principal.server.address.com" 
```

Comment:
I know this just a dummy, but server is always specified without protocol prefix.

Reply:
Updated, server will have address without mentioning protocol prefix.